src/main/resources/application.properties

```diff
@@ -1,10 +1,7 @@
 spring.datasource.url=jdbc:postgresql:sandogh
-spring.datasource.username=sandogh
-spring.datasource.password=sandogh
+spring.datasource.username=admin
```

Comment:
I don't like to have these kinds of changes. 
Do not commit the changes according to your test/development machine configurations.
You can use profiling to customize the configuration based on your machine.
READ THEM AND APPLY.